config/yugabyte/regression_pipelines/scan_workloads/postgres/scanG12_bitmapscan_orCondition.yaml

```diff
@@ -0,0 +1,112 @@
+            - CREATE INDEX ON pkey_for_bitmapscan_tbl_1 (col_bigint_2 ASC);
+            - CREATE INDEX ON pkey_for_bitmapscan_tbl_1 USING HASH (col_bigint_3);
+        cleanup:
+            - DROP TABLE IF EXISTS pkey_for_bitmapscan_tbl_1;
+        loadRules:
+            - table: pkey_for_bitmapscan_tbl_
+              count: 1
+              rows: 1000000
+              columns:
+                  - name: col_bigint_id_
+                    count: 1
+                    util: PrimaryIntGen
+                    params: [1, 1000000]
+                  - name: col_bigint_
+                    count: 5
+                    util: RandomNumber
+                    params: [1, 1000000]
+                  - name: col_float2_
+                    count: 2
+                    util: RandomNoWithDecimalPoints
+                    params: [1, 1000000, 2]
+                  - name: col_float5_
+                    count: 2
+                    util: RandomNoWithDecimalPoints
+                    params: [1, 1000000, 5]
+                  - name: col_boolean_
+                    count: 1
+                    util: RandomBoolean
+                  - name: col_varchar10_
+                    count: 1
+                    util: RandomAString
+                    params: [10, 10]
+                  - name: col_varchar100_
+                    count: 2
+                    util: RandomAString
+                    params: [100, 100]
+                  - name: col_varchar500_
+                    count: 1
+                    util: RandomAString
+                    params: [500, 500]
+        executeRules:
+            - workload: scanG12_bitmapscan_orCondition_10row
+              run:
+                  - name: pkey_INclause_rangescan_increasing_rows_10row_test
+                    weight: 100
+                    queries:
+                        - query: select col_bigint_id_1, col_bigint_1 from pkey_for_bitmapscan_tbl_1 where col_bigint_id_1 <= 3 OR col_bigint_2 <= 3 OR col_bigint_3 IN (?,?,?)
+                          bindings:
+                              count: 3
+                              util: RandomNumber
+                              params: [ 1, 1000000 ]
+            - workload: scanG12_bitmapscan_orCondition_100row
+              run:
+                  - name: pkey_INclause_rangescan_increasing_rows_100row_test
+                    weight: 100
+                    queries:
+                        - query: select col_bigint_id_1, col_bigint_1 from pkey_for_bitmapscan_tbl_1 where col_bigint_id_1 <= 33 OR col_bigint_2 <= 33 OR col_bigint_3 IN (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)
```

Comment:
Optional: It could be good to have one query where the select targets are only the pkey and another query where no select targets are in the pkey as we could optimize the former case in the future.

Also, it could be good to have a query where there is no bitmap scan on the pkey index. This is also because we might be able to optimize the case where there is a bitmap scan on the pkey index.